components/builder-depot/src/server.rs

```diff
@@ -2623,6 +2690,10 @@ mod test {
         package.set_config("config".to_string());
         package.set_target("x86_64-linux".to_string());
 
+        let mut channels = protobuf::RepeatedField::new();
+        channels.push("unstable".to_string());
+        package.set_channels(channels);
```

Comment:
@christophermaier Every package is guaranteed to always be in the `unstable` channel.